tests/api/test_group_api.py

```diff
@@ -4,15 +4,15 @@
 import requests
 from fauxfactory import gen_string
 from insights.config import Settings
+from insights.session import Session
 logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
 
 
 class GroupsAPI(unittest.TestCase):
     def setup_class(self):
         self.setting = Settings()
-        self.session = requests.session()
-        self.session.cert = self.setting.get_certs()
-        self.session.verify = False
+        self.session_instance = Session() 
```

Comment:
@rohitkadam19 this will only be reducing the lines of code but not solve the purpose of creating one session and then proceeding with all the tests. Every time a new session will be created in this way for each test which is something we should be avoiding.